src/components/MyKiva/BadgeCompleted.vue

```diff
@@ -110,16 +106,12 @@ const badgeImage = computed(() => {
 });
 
 const badgeCategory = computed(() => badge.value?.challengeName ?? '');
-const tiers = computed(() => lendingAchievement.value?.tiers ?? []);
-
-const sortedTiers = computed(() => {
-	const defaultTiers = [...tiers.value];
-	return defaultTiers.sort((a, b) => b.target - a.target);
-});
+const tiers = computed(() => badge.value?.tiers ?? []);
```

Comment:
With some of my changes today, this is `badge.value?.achievementData?.tiers`

Reply:
Unfortunately that method won't work here, since the current tier is the current active tier. You could add a new method for last completed tier.